spec/models/car_tax_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```

Comment:
https://github.com/thoughtbot/shoulda-matchers this gem should be used for testing associations and validations